tests/math_builtin_api/modules/test_generator.py

```diff
@@ -52,6 +54,28 @@
                     "$FUNCTION_CALL"
                     "}, $DATA, ref, refPtr$ACCURACY);\n}\n") }
 
+function_call_template = Template("""
```

Comment:
Quite clearer!
Now, if you put each template declaration just before each function using it, it will be perfect.

Reply:
Done.